lib/HeroBonus.cpp

```diff
@@ -763,6 +763,24 @@ void CBonusSystemNode::popBonuses(const CSelector &s)
 		child->popBonuses(s);
 }
 
+void CBonusSystemNode::updateBonuses(const CSelector &s)

```

Comment:
Is selector actually make sense here? What are use-cases of this method with custom selector?